services/app-api/src/postgres/contractAndRates/submitContract.ts

```diff
@@ -100,117 +91,6 @@ export async function submitContract(
                 },
             })
 
-            // OLD SUBMIT STYLE
-            // Update the contract to include the submitInfo ID
-            const updated = await tx.contractRevisionTable.update({
-                where: {
-                    id: currentRev.id,
-                },
-                data: {
-                    submitInfo: {
-                        connect: {
```

Comment:
we probably still need this part of the call, setting the submitInfo, but maybe nothing else